### ISSUE TITLE
[Game Instability] Dredge

### ISSUE THREAD

**Issue by rmaia** [CryoByte33/steam-deck-utilities#130]
hi, let me know if this is not the best place to bring it up, but I wanted to report I've been having some compatibility issues with the game Dredge on the Steam Deck when using CryoUtilities recommended settings. The game will randomly and repeatedly freeze for 5-15 seconds, and come back with high GPU utilization for another second or so. Reverting the Cryoutilities settings to stock makes this behavior go away. Tested turning settings from recommended to stock 3x, consistently getting these results - freezes when using recommended settings, plays fine when using stock settings. 

I haven't been too scientific about it but it seems to be related to the settings under the "Memory" tab. Keeping VRAM at 4GB and recommended swap file size/swappiness seem to be fine. But I haven't gone through every permutation of memory and swap settings (let alone values of different settings) to find a root cause. 

Sharing mostly for visibility, hope this helps, thanks for the great tool!

**Comment by CryoByte33** [comment]
This is the first I've heard of the issue, do you have the same issue if you re-enable CU's recommended settings and do a reboot?

**Comment by rmaia** [comment]
I believe so. I'll give it another try to confirm. Could just be my system too, I guess. I could try recording a little video showing what I'm running into, it's pretty consistent so easy for me to replicate, just let me know!

**Comment by rmaia** [comment]
Restarting didn't seem to help, also started getting `amdgpu: ring gfx_0.0.0 timeout` errors. Reverted to stock and restarted, have been playing for a handful of hours now without crashes 🤞

**Comment by CryoByte33** [comment]
Very interesting. I don't have the game so I can't test, but nothing CU does should cause amdgpu to throw errors 🤔

Maybe try enabling everything except huge pages?

**Comment by rmaia** [comment]
interesting. maybe it's something with my system. The freezing would happen
even just sitting on the starting menu, so it wasn't related to anything I
was doing. I finished the game a couple weeks ago after turning the changes
off, so I played it for several hours without an issue… I turned the
cryoutilities recommended settings back on after I finished (so I could
play Marvel's Midnight Suns), so I tried Dredge again just now to test real
quick, still getting the freezing/stuttering. Really weird!

**Comment by chrismcmacken** [comment]
Hmm, which model deck did you buy? Any other changes?

**Comment by rmaia** [comment]
256GB model, no other changes other than some aesthetic decky plugins

**Comment by rmaia** [comment]
yep I tried reinstalling and verifying the integrity of local files. only played handheld!
